scripts/cibuild

```diff
@@ -17,10 +17,13 @@ if [[ "${BASH_SOURCE[0]}" == "$0" ]]; then
     if [[ "${1:-}" == "--help" ]]; then
         usage
     else
-        docker build \
+        DOCKER_BUILDKIT=1 docker build \
+	    --build-arg BUILDKIT_INLINE_CACHE=1 \
+	    --build-arg BUILD_TYPE=fullbuild \
             --platform linux/amd64 \
             --build-arg CUDA_VERSION="12.1.1" \
             --build-arg UBUNTU_VERSION="22.04" \
+	    --cache-from=quay.io/azavea/raster-vision:latest \
```

Comment:
I'm seeing this in the CI log:
![image](https://github.com/azavea/raster-vision/assets/13014700/2957f370-52db-4b44-b76c-28bc06a35544)

Probably because the correct tag is `pytorch-latest`.

Reply:
:heavy_check_mark: Will correct in a subsequent push to this branch.